chains/solana/contracts/programs/ccip-router/src/lib.rs

```diff
@@ -18,20 +13,14 @@ use crate::state::*;
 mod event;
 use crate::event::*;
 
-mod merkle;
-use crate::merkle::*;
-
 mod messages;
 use crate::messages::*;
 
-mod pools;
-use crate::pools::*;
-
 mod ocr3base;
 use crate::ocr3base::*;
 
-mod fee_quoter;
-use crate::fee_quoter::*;
+mod instructions;
+use crate::instructions::*;
 
 // Anchor discriminators for CPI calls
 const CCIP_RECEIVE_DISCRIMINATOR: [u8; 8] = [0x0b, 0xf4, 0x09, 0xf9, 0x2c, 0x53, 0x2f, 0xf5]; // ccip_receive
```

Comment:
Is this var then imported from the instructions crate?

Reply:
Yes, the new `offramp.rs` file imports this. All the `const`s defined in the `lib.rs` are used in either the onramp or the offramp now, but they are tied to our interface with receivers and tokenpools so the constants themselves shouldn't be versioned.